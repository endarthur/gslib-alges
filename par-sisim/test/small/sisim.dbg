
 Category  number  1 =        1.000
           global prob value =   0.1470
           number of structures =   1
           nugget effect        =   0.0500
           type of structure   1 =   1
           aa parameter         =     100.0000
           cc parameter         =       0.9500
           ang1, ang2, ang3     =   0.00  0.00  0.00
           anis1, anis2         =       1.0000      0.2500

 Category  number  2 =        2.000
           global prob value =   0.6880
           number of structures =   1
           nugget effect        =   0.0500
           type of structure   1 =   1
           aa parameter         =     100.0000
           cc parameter         =       0.9500
           ang1, ang2, ang3     =   0.00  0.00  0.00
           anis1, anis2         =       1.0000      0.2500

 Category  number  3 =        3.000
           global prob value =   0.1650
           number of structures =   1
           nugget effect        =   0.0500
           type of structure   1 =   1
           aa parameter         =     100.0000
           cc parameter         =       0.9500
           ang1, ang2, ang3     =   0.00  0.00  0.00
           anis1, anis2         =       1.0000      0.2500

  Data for SISIM: Variable number  7
  Number of acceptable data  =     2375
  Equal Weighted Average     =       2.0143
  Equal Weighted Variance    =       0.3114


 There are          317  nearby nodes that will be 
 checked until enough close data are found.

