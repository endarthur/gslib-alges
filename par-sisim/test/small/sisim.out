SISIM SIMULATIONS:  Sondajes de exploracion                                     
 1   40   60   12  5.0000000      5.0000000      6.0000000      10.0000      10.0000      12.0000      16
Simulated Value
